actors/account/tests/account_actor_test.rs

```diff
@@ -158,6 +159,46 @@ fn authenticate_message() {
         .unwrap());
 }
 
+#[test]
+fn accept_arbitrary() {
```

Comment:
Nit: arbitrary is a small misnomer since there is a condition. (Greater than the private range)

Reply:
@vyzo address the nit?

Reply:
it is test_fallback now.